Dockerfile

```diff
@@ -1,8 +1,7 @@
 FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 AS builder
 WORKDIR /go/src/sigs.k8s.io/cluster-api-provider-aws
 COPY . .
-RUN go build -o ./machine-controller-manager ./cmd/manager
-RUN go build -o ./manager ./vendor/github.com/openshift/cluster-api/cmd/manager
+RUN NO_DOCKER=1 make build
```

Comment:
how is this related to this PR/commit?

Reply:
https://github.com/openshift/cluster-api-provider-aws/pull/179#issuecomment-473830832

Reply:
Thanks @ingvagabund, yeah it basically removes duplicate logic by just running the build trough make for the image release process.